extensions/emmet/src/test/wrapWithAbbreviation.test.ts

```diff
@@ -340,6 +340,54 @@ suite('Tests for Wrap with Abbreviations', () => {
 			});
 		});
 	});
+
+	test('Wrap maintains href', () => {
+		const contents = `
+			www.google.com
+		`;
+		const expectedContents = `
+			<a href="www.google.com">www.google.com</a>
+		`;
+
+		return withRandomFileEditor(contents, 'html', (editor, _) => {
+			editor.selections = [new Selection(1, 1, 1, 1)];
+			const promise = wrapWithAbbreviation({ abbreviation: 'a' });
+			if (!promise) {
+				assert.equal(1, 2, 'Wrap with Abbreviation returned undefined.');
+				return Promise.resolve();
+			}
+
+			return promise.then(() => {
+				assert.equal(editor.document.getText(), expectedContents);
+				return Promise.resolve();
+			});
+		});
+	});
+
+	test('Wrap Individual Lines maintains href', () => {
+		const contents = `
+			www.google.com
+			www.google.com.au
+		`;
+		const expectedContents = `
+			<a href="www.google.com">www.google.com</a>
+			<a href="www.google.com.au">www.google.com.au</a>
+		`;
+
+		return withRandomFileEditor(contents, 'html', (editor, _) => {
+			editor.selections = [new Selection(1, 0, 1, 17), new Selection(2, 0, 2, 20)];
+			const promise = wrapIndividualLinesWithAbbreviation({ abbreviation: 'a' });
```

Comment:
To test the individual lines variation, change the editor.selections to an array with single item `new Selection(1, 0, 2, 20)` and change the abbreviation to `a*`.

After this change, you will see that emmet adds the `http://` to the value in the href attribute

Reply:
Ah, understood. Updated the expected results, see the latest commit.